rdt-open-reader/src/main/java/io/ona/rdt/activity/LoginActivity.java

```diff
@@ -27,7 +25,7 @@ public void onCreate(Bundle savedInstanceState) {
         updateLocale(this);
         updateFTSDetails();
         super.onCreate(savedInstanceState);
-        addAttributionText();
+        updateLocale(this);
```

Comment:
Any reason we need this here? It's already called https://github.com/onaio/rdt-standard/pull/930/files#diff-e48c6bd6ae1616b9ca15983c55d645d4dc31c4e5f1bdcdbdad5261b4501c0ecbR25.

Reply:
Not sure, just keeping the existing code. It was called inside addAttributionText().

Reply:
Ok, that was needed for the attribution text to show correctly (be translated). It can be removed.